packages/node_modules/@webex/plugin-meetings/test/unit/spec/sparkTestUsers.js

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
is this file necessary? i feel that we already have a spark test users we could import? or this could be placed elsewhere

Reply:
The other file was not doing mercury connect . I can revisit again to see if we can use one